### PR TITLE
Fix the cluster spec path of master to launch workers.

### LINKED ISSUE #2168: How to use  `--cluser_spec`
Need to push spec.py in docker image when using elasticdl client, but also need it when submitting a job.

### PR DESCRIPTION
**Pull request by workingloong**
Fix #2168
The path of the cluster spec is fixed in the image. We cannot pass `args.cluster_spec` to master.

### CODE REVIEW

File: elasticdl_client/api.py [hunk 1]
```diff
@@ -62,14 +65,15 @@ def init_zoo(args):
...
-COPY ./{{ CLUSTER_SPEC_NAME }} /cluster_spec/{{ CLUSTER_SPEC_NAME }}\
+COPY ./{{ CLUSTER_SPEC_NAME }} {{CLUSTER_SPEC_PATH}}/{{ CLUSTER_SPEC_NAME }}\
 {% endif %}
 """
     template = Template(tmpl_str)
     docker_file_content = template.render(
         BASE_IMAGE=args.base_image,
         EXTRA_PYPI_INDEX=args.extra_pypi_index,
         CLUSTER_SPEC_NAME=cluster_spec_name,
+        CLUSTER_SPEC_PATH=ClusterSpecConfig.CLUSTER_SPEC_DIR,
```
Comment (brightcoder01) [inline elasticdl_client/api.py]: CLUSTER_SPEC_PATH -> CLUSTER_SPEC_FOLDER_PATH

Comment (workingloong) [reply elasticdl_client/api.py]: CLUSTER_SPEC_DIR ?

Comment (brightcoder01) [reply elasticdl_client/api.py]: > CLUSTER_SPEC_DIR ?

This name is better

**Review by skydoorkai — APPROVED** [review]

**Comment by skydoorkai** [comment]
Also need to update on-prem tutorial.

**Pull request merged**